plugins/qeta-backend/src/database/QetaStore.ts

```diff
@@ -91,6 +91,15 @@ export interface TagResponse {
   tag: string;
   questionsCount: number;
 }
+export interface Attachment {
+  id: number;
+  locationType: string;
+  locationUri: string;
+  binaryImage: Buffer;
```

Comment:
I think this also requires the mimeType of the file to be able to show those properly later